languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -26988,7 +26960,7 @@ Andere Projekte</example>
                 </marker>
             </pattern>
             <message>Wenn '<match no="2"/>' hier als Nomen genutzt wird, muss es großgeschrieben werden.</message>
-            <example correction="">Er müsste sich aufs <marker>lernen</marker> konzentrieren.</example>
+            <example correction="Lernen">Er müsste sich aufs <marker>lernen</marker> konzentrieren.</example>
```

Comment:
Das klappt so noch nicht: wenn es in der `<message>` keine `<suggestion>` gibt, darf auch das `correction`-Attribut nicht belegt werden, sonst gehen die Tests nicht. Das kannst Du mit `./build.sh de test` testen.